packages/block-library/src/query-pagination/block.json

```diff
@@ -23,7 +23,15 @@
 		"html": false,
 		"color": {
 			"gradients": true,
-			"link": true
+			"link": true,
+			"__experimentalDefaultControls": {
+				"background": true,
+				"text": true,
+				"__experimentalDefaultControls": {
```

Comment:
This nested object looks like a typo 😀

Reply:
Thanks. Must have been running out of seem by the end all these copy and pastes 🙂